src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
```

Comment:
@yuya-oc Help welcome. I don't think it's possible to find an alternative shortcut that works on every layout as this is a alt shortcut for that is supposed to be here for UX purpose; tell me if I'm wrong.